skimage/feature/texture.py

```diff
@@ -291,3 +295,154 @@ def local_binary_pattern(image, P, R, method='default'):
     image = np.ascontiguousarray(image, dtype=np.double)
     output = _local_binary_pattern(image, P, R, methods[method.lower()])
     return output
+
+
+def multiblock_lbp(int_image, x, y, width, height):
+    """Multi-block local binary pattern.
+
+    The features are calculated similarly to local binary patterns (LBPs),
+    except that summed blocks are used instead of individual pixel values.
+
+    MB-LBP is an extension of LBP that can be computed on multiple scales
+    in constant time using the integral image.
```

Comment:
Wrap to 79 chars per line, and start the next sentence with "Nine" instead of the numeral 9.